packages/studio-be/src/studio/cloud/schemas.ts

```diff
@@ -0,0 +1,6 @@
+import { z } from 'zod'
+
+export const DeployRequestSchema = z.object({
```

Comment:
sadly we use joi everywhere in this project. as a matter of consistency, can you rewrite this schema in with joi (if you have time)

Reply:
![Screen Shot 2022-08-08 at 10 14 54 AM](https://user-images.githubusercontent.com/892367/183438853-f1e69c6d-1311-486a-bfbc-d6bd220490c5.png)
We use joi in only a handful of files. We can migrate to zod gradually. They can live together peacefully during this period of transition.

Reply:
Joi is used litterally in every codebase related to botpress... I strongly recomment keeping Joi and not add another library that everyone will have to learn when interacting with this code

Reply:
Not anymore. All new codebases in the company in the last few months are uzing Zod. In a Typescript codebase, zod is clearly better than joi due to it's type inference feature. I think we should move to Zod, and we can do it incrementally.

Reply:
I'll revert to joi for now

Reply:
Fixed in efea418